litellm/tests/test_prompt_factory.py

```diff
@@ -212,7 +212,7 @@ def test_convert_url_to_img():
     [
         ("data:image/jpeg;base64,1234", "image/jpeg"),
         ("data:application/pdf;base64,1234", "application/pdf"),
-        ("data:image\/jpeg;base64,1234", "image/jpeg"),
```

Comment:
@msabramo this is actually an intended test - i didn't pick this issue on flake8. 

This is testing the functions ability to handle malformed b64 strings